pkgs/misc/uboot/default.nix

```diff
@@ -71,6 +74,12 @@ let
     ];
     depsBuildBuild = [ buildPackages.stdenv.cc ];
 
+    buildInputs = [
```

Comment:
Given that these are used for tools, I think they should be nativeBuildInputs? This matters for cross-compilation, e.g. when compiling aarch64 binaries from an amd64 builder you'd want to have the amd64 version of those libs, not the aarch64 version.